NEWS

```diff
@@ -1,5 +1,5 @@
-Flocker 0.4.0pre1 (2015-03-24)
-==============================
+Flocker 0.4.0 (2015-04-07)
+==========================
 
 - New getting started guide. (FLOC-1261)
 - Updated to Twisted 15.0. (FLOC-1302)
```

Comment:
I wonder if this is relevant to anybody, since we don't expose the fact that we are written in python.